.travis.yml

```diff
@@ -16,11 +16,13 @@ before_script:
   - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
         rustup component add --toolchain nightly rustfmt-preview;
         which rustfmt || cargo install --force rustfmt-nightly;
+        rustup component add clippy;
```

Comment:
Does it matter that not all nightly versions support clippy? Can the build fail if thats the case (quite common: https://rust-lang.github.io/rustup-components-history/index.html)

Reply:
Can we make a stable job that only runs clippy? Or will that fail due to use of nightly features?

Reply:
https://github.com/rust-lang/rust-clippy/#travis-ci